### PR TITLE
Add a config to enable seeing average ns per tick in waila

### PR DESCRIPTION
**Pull request by BlueWeabo**
Shows the CPU load (same as scanner) on WAILA tooltips of Multiblock controllers.
image

Disabled by default as most people likely don't care.

### CODE REVIEW

**Review by S4mpsa — APPROVED** [review]
Tested on server & code looks fine, but someone else might want to merge this as it's a feature addition and I am biased by requesting this myself.

**Comment by minecraft7771** [comment]
Where unit

**Comment by BlueWeabo** [comment]
> Where unit

what?

**Comment by minecraft7771** [comment]
It doesnt tell you that it's nano seconds

**Comment by Dream-Master** [comment]
Maybe we can add ns to tooltip ?

File: src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_MultiBlockBase.java [hunk 1]
```diff
@@ -1659,7 +1659,11 @@ public void getWailaBody(ItemStack itemStack, List<String> currentTip, IWailaDat
         }
         currentTip.add(
             GT_Waila.getMachineProgressString(isActive, tag.getInteger("maxProgress"), tag.getInteger("progress")));
-
+        // Show ns on the tooltip
+        if (GT_Mod.gregtechproxy.wailaAverageNS) {
+            int tAverageTime = tag.getInteger("averageNS");
+            currentTip.add("Average CPU load of ~" + GT_Utility.formatNumbers(tAverageTime) + "ns");
```
Comment (minecraft7771) [inline src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_MultiBlockBase.java:1665]: There needs to be a space between number and unit :trollface:

**Comment by miozune** [comment]
Add lang entry for config
ns

**Comment by Pilad** [comment]
> Add lang entry for config ns

Where is this menu?

**Comment by Pilad** [comment]
Found. I thought it was in the WAILAPlugins menu, which is called through the 0 button.

**Pull request merged**